co2eq/car/cars.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Can you turn this into a list? It doesn't make sense to have an object with keys that contain the same information as the content itself.
I suggest:
```json
{
  "footprints": [
    { "size": null, ... },
    { ... }
  ]
}
```

Reply:
Thank you for reviewing. In that case, should carbonIntensity (kgCO2/km) be in a function in index.js, separate from the list in cars.json?

Reply:
I will put data on carbon intensity in cars.json, sorry for the confusion.

Reply:
I still think you can have one carbonIntensity per entry, i.e.
`{ "size": null, ... "carbonIntensity": 0.53 }`

Reply:
I agree, thank you for your patience.